examples/main/main.cpp

```diff
@@ -298,7 +300,7 @@ int main(int argc, char ** argv) {
         }
 
         // remove any "future" tokens that we might have inherited from the previous session
-        llama_kv_cache_seq_rm(ctx, -1, n_matching_session_tokens, -1);
+        llama_kv_cache_tokens_rm(ctx, n_matching_session_tokens, -1);
```

Comment:
The old version is the correct one - probably this is an artifact from resolving conflicts